code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1505,3 +1505,46 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	M.satiety += 5 //for context, vitamins give 30 satiety per tick
 	..()
 	. = TRUE
+
+/datum/reagent/consumable/ethanol/fringe_weaver
+	name = "Fringe Weaver"
+	id = "fringe_weaver"
+	description = "Bubbly, classy, and undoubtedly strong - a Glitch City classic."
+	color = "#FFEAC4"
+	boozepwr = 90 //classy hooch, essentially, but lower pwr to make up for slightly easier access
+	taste_description = "ethylic alcohol with a hint of sugar"
+	glass_icon_state = "fringe_weaver"
+	glass_name = "Fringe Weaver"
+	glass_desc = "It's a wonder it doesn't spill out of the glass."
+
+/datum/reagent/consumable/ethanol/sugar_rush
+	name = "Sugar Rush"
+	id = "sugar_rush"
+	description = "Sweet, light, and fruity - as girly as it gets."
+	color = "#FF226C"
+	boozepwr = 10
+	taste_description = "your arteries clogging with sugar"
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	glass_icon_state = "sugar_rush"
+	glass_name = "Sugar Rush"
+	glass_desc = "If you can't mix a Sugar Rush, you can't tend bar."
+
+/datum/reagent/consumable/ethanol/sugar_rush/on_mob_life(mob/living/M)
+	M.satiety -= 10 //junky as hell! a whole glass will keep you from being able to eat junk food
+	..()
+	. = TRUE
+
+/datum/reagent/consumable/ethanol/crevice_spike
+	name = "Crevice Spike"
+	id = "crevice_spike"
+	description = "Sour, bitter, and smashingly sobering."
+	color = "#5BD231"
+	boozepwr = -10 //sobers you up - ideally, one would drink to get hit with brute damage now to avoid alcohol problems later
+	taste_description = "a bitter SPIKE with a sour aftertaste"
+	glass_icon_state = "crevice_spike"
+	glass_name = "Crevice Spike"
+	glass_desc = "It'll either knock the drunkenness out of you or knock you out cold. Both, probably."
+
+/datum/reagent/consumable/ethanol/crevice_spike/on_mob_add(mob/living/L)
+	L.adjustBruteLoss(15) //That first gulp hurts going down! But you can finish the drink much easier on your gut-zone.
```

Comment:
This is going to be abused to an insane degree, do we have any other chems that do straight damage like this?

Reply:
Would it be better to reduce the damage on first ingestion (it only deals damage the first time it enters your system, and won't again until its out), or find some other means of stunning / harming the drinker to offset the sobering effect?

Reply:
remember it's only on the first /sip/ of it. Not certain how it could be abused to an insane degree, as its only useful once per person (until it metabolizes out i guess)

Reply:
@MrDoomBringer when I was testing this (and figuring out the code) I accidentally made it do 2 brute damage per tick while it metabolized - suffice to say, it killed the drinker really fast.

Reply:
I think the primary concern is that people might inject 0.1 units at a time, once every two seconds, to deal damage a lot faster than any conventional chem. Considering the finnicky-ness of doing something like that, though, I doubt it'll be too concerning.

Reply:
@fludd12 I feel that if you've got someone in a position where you can inject them repeatedly with crevice spike over time without them noticing / panicking, you've got a person in a position where you can do just about anything to them.

Reply:
My exact same thought on the subject!

Reply:
I meant it more as a cheap additive for syringe gun mixes, but I think the new change fixes it.